src/molecules/Product.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+
+function Product() {
+  const [items, setItems] = useState([
+    {
+      id: 1,
+      name: "item 1",
+      description: "item 1 description will go somewhere here",
+      image:
+        "https://i.pinimg.com/736x/52/bc/39/52bc3928fd63daa22ebfb555f9ae07dd.jpg",
+    },
+    {
+      id: 2,
+      name: "item 2",
+      description: "item 2 description will go somewhere here",
+      image: "image 2 will be here",
+    },
+    {
+      id: 3,
+      name: "item 3",
+      description: "item 3 description will go somewhere here",
+      image: "image 3 will be here",
+    },

```

Comment:
each object of this array should be passed as argument to Product component.
This should happen from Products organism, which would be responsible for showing all of them together